static/src/stylesheets/module/site-messages/_first-pv-consent.scss

```diff
@@ -152,6 +177,8 @@ $btn-height: ($gs-baseline*3.5);
 .site-message--first-pv-consent__actions {
     @include fs-textSans(3);
     margin-top: $gs-baseline * 2;
+    font-weight: bold;
+    font-size: 16px;
```

Comment:
could this be using the correct `fs-textSans()` instead? (think it's 5 or 6)

Reply:
done, thanks for the tip